pkg/scheduler/placement/fixed_rule_test.go

```diff
@@ -62,7 +62,7 @@ func TestFixedRule(t *testing.T) {
 
 func TestFixedRulePlace(t *testing.T) {
 	// Create the structure for the test
-	data := `
+	const data = `
```

Comment:
we should move this to `testrule_test.go` as other occurrences are using the method `testrule_test#initQueueStructure`